src/components/Signup/Signup.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Joi from 'joi-browser';
+import PropTypes from 'prop-types';
+import signupInputSchema from '../../schemas/signup';
+import signupAction from '../../actions/user/signupAction';
+import Form from '../Form/Form';
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+import GoogleIcon from '../../../assets/icon/google.png';
```

Comment:
pls use an index file for this, consult @LukasChiama for this

Reply:
I consulted Chukwualuka and implemented the necessary changes.